tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1017,13 +1017,13 @@ def test_copy_non_existent_file_fails(ansible_zos_module, is_remote):
 
 @pytest.mark.uss
 @pytest.mark.parametrize("src", [
-    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=False),
-    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=True),])
-def test_copy_dest_directory_remain(ansible_zos_module, src):
+    dict(src="/etc/profile", is_remote=False),
+    dict(src="/etc/profile", is_remote=True),])
+def test_ensure_copy_file_dont_change_permission_on_dest(ansible_zos_module, src):
     hosts = ansible_zos_module
     dest_path = "/tmp/test/"
     try:
-        hosts.all.shell(cmd="mkdir -p {0}".format(dest_path))
+        hosts.all.file(path=dest_path, state="directory")
         hosts.all.shell(cmd="chmod 750 {0}".format(dest_path))
```

Comment:
You can add `mode: "750"` to the `file` task above to set the permissions for the directory. Same for the other test.